pycaret/utils/generic.py

```diff
@@ -488,6 +488,8 @@ def np_list_arange(
     return range_
 
 
+# Again we dont have ground truth in clustering, just an educated guess on what
```

Comment:
See [here](https://github.com/pycaret/pycaret/blob/master/pycaret/containers/metrics/clustering.py) how metrics are set up for non-time series clustering. I recommend following a similar structure for time series clustering as well.

We can add new metrics file for time series clustering in the same folder (`containers.metrics`). Similar to models, we will have to rename the existing `time_series.py` to `time_series_forecasting.py`